src/index.js

```diff
@@ -72,12 +66,7 @@ function render(
     unmount: () => wrapper.unmount(),
     html: () => wrapper.html(),
     emitted: () => wrapper.emitted(),
-    rerender: ({props}) => {
-      wrapper.unmount()
-      mountedWrappers.delete(wrapper)
-
-      wrapper = mountComponent(TestComponent, props)
-    },
+    rerender: props => wrapper.setProps(props),
```

Comment:
This is mostly it. Other changes are because we don't need a `wrapper` variable (and a `mountComponent` function) anymore.